### PR TITLE
Feature/webpack 5 sb web

### PR DESCRIPTION
**Pull request by MKaHead**
- Upgrade Storybook 6.5
- Upgrade to webpack 5 and do migration
- Remove watchConfig polling: main reason for poor storybook performance under windows
- Use @storybook/web-components instead of html
- Use terser instead of uglify for dist build
- Remove hack to hide the new version banner instead use a flag
- Use main.js config to set static path for storybook
- Some more dependencies upgrades and remove obsolete packages

# Merge Checklist:
- [ ] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [ ] Tests are sufficient.
- [ ] Documentation is up to date.
- [ ] Changelog is up to date.
- [ ] Typescript typings for properties are up to date.
- [ ] Designers approved changes or no approval needed.
- [ ] Dependencies to other components still work.

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import '../src/components/05-utils/polyfill/index';
-import { addParameters } from '@storybook/html';
+// import '../src/components/05-utils/polyfill/index';
```
Comment (markus-walther) [inline .storybook/preview.js]: Don't comment out stuff, just delete it - it's the job of version-control systems (i.e. git for us) to recover old code from a previous version

File: package.json [hunk 2]
```diff
@@ -76,33 +75,31 @@
...
+    "rollup-plugin-copy": "^3.4.0",
+    "rollup-plugin-sass": "^1.2.12",
+    "rollup-plugin-terser": "^7.0.2",
+    "sass": "1.51.0",
+    "sass-loader": "^12.6.0",
+    "svgo": "^2.8.0",
     "testcafe": "1.9.2",
-    "testcafe-browser-provider-browserstack": "1.12.0",
-    "to-string-loader": "^1.1.5"
+    "webpack": "^5.72.1"
```
Comment (markus-walther) [inline package.json:102]: good!

File: scripts/build/dist.rollup.js [hunk 3]
```diff
@@ -1,13 +1,12 @@
 const fs = require('fs');
 const { extname } = require('path');
-const { uglify } = require('rollup-plugin-uglify');
+const { terser } = require('rollup-plugin-terser');
```
Comment (markus-walther) [inline scripts/build/dist.rollup.js:3]: good!

File: scripts/build/dist.rollup.js [hunk 4]
```diff
@@ -59,17 +58,13 @@ const dist = {
       mainFields: ['module', 'main', 'browser'],
     }),
     commonjs({
-      namedExports: {
-        '@skatejs/val': ['val'],
-      },
       include: /node_modules/, // needed because we are in a monorepo and the node_modules could be somewhere up or in the root
     }),
     babel({
```
Comment (markus-walther) [inline scripts/build/dist.rollup.js:63]: one of the next PRs: get rid of babel!

File: scripts/build/dist.rollup.js [hunk 5]
```diff
@@ -86,7 +81,11 @@ const dist = {
         '../../../../node_modules/@babel/**',
       ],
     }),
-    uglify(),
+    terser({
+      ecma: 5,
```
Comment (markus-walther) [inline scripts/build/dist.rollup.js:85]: one of the next PRs: ES6!

**Review by markus-walther — CHANGES_REQUESTED** [review]
Hi Marco,

small findings only.

I am torn between wanting the flaky tests deleted, or commented out via multi-line comments, or left like you commented them out with single-line comments.

In general I think commenting out is the wrong tool for the job in the age of version-control systems.

Maybe compromise by adding a link to the conversion-to-Playwright issue (you didn't create it yet).

I leave it to you to make a reasoned decision here.

Rest is fine. Good work!

Markus

**Comment by MKaHead** [comment]
@markus-walther Create issue #2284 and delete flaky tests.

**Review by markus-walther — APPROVED** [review]
Marco,

good work indeed!

Playwright has delay functions, no need to do setTimeout thingies client-side. But you can clean that up in your footer test another PR.

Approved!

--Markus

**Pull request merged**